webpack.config.js

```diff
@@ -119,13 +119,13 @@ module.exports = {
         test: /\.css$/,
         include: [
           path.resolve(__dirname, './node_modules/monaco-editor'),
-          path.resolve(__dirname, './node_modules/@patternfly'),
+          path.resolve(__dirname, './node_modules/@open-cluster-management/ui-components'),
```

Comment:
Any reason why this change is needed?

Reply:
@ycao56 the style got messed up when i updated ui-components. I assumed there was a style change in that repo, and it shouldn't change anything in grc since ui-components should include the patternfly styles

Reply:
ui-components does not import any styles. that's why we include `@patternfly` styles explicitly and skip any styles from `@open-cluster-management/ui-components`